### ISSUE TITLE
Allow setting of highlight focus day

### ISSUE THREAD

**Issue by jdhines** [ng-bootstrap/ng-bootstrap#2136]
Please remember, the issues forum is __NOT__ for support requests. It is for bugs and feature requests only.
Please read https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CONTRIBUTING.md and search
existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.

### Bug description:
When the datepicker popup is launched, the first day of the month is highlighted, whereas convention seems to indicate that the current day should be highlighted

###Expected behavior:
The .btn-light class should be applied to the current date by default, or there should be options added to the API such as `[highlightDay]="first|current"`.

### Link to minimally-working plunker that reproduces the issue:
https://ng-bootstrap.github.io/app/components/datepicker/demos/popup/plnkr.html

### Version of Angular, ng-bootstrap, and Bootstrap:
ng-bootstrap: v. 1.0.0.-beta-9

**Comment by jdhines** [comment]
Yes, the keyboard navigation focus. If no date has been picked and the popup is opened, the only thing highlighted is the first day of the month. I was just mentioning that in any other datepicker I've seen, if anything is highlighted, it's the current day, so that if that's what the user wants, it's easy to pick. 
See the datepickers from [jQuery](https://jqueryui.com/datepicker/), [Angular Material](https://material.angular.io/components/datepicker/overview), and [Angular datepicker](https://720kb.github.io/angular-datepicker/)

And if a day _has_ been selected, it seems weird to me when the picker opens, both the selected day and the first day of the month are highlighted (albeit with different colors, but one shade of blue doesn't automatically mean something different than another shade of blue). I've seen [datepickers](https://www.jqwidgets.com/jquery-widgets-demo/demos/jqxdatetimeinput/index.htm#demos/jqxdatetimeinput/defaultfunctionality.htm) where today's date is always highlighted, but none others where the first of the month is.

Maybe it's just me, but I would prefer there to be no highlight whatsoever if the current value is null, at least until the user hits an arrow key, unless the highlight is the current day. 

So maybe if there could be a new option for "keyboardFocusHighlight"?

**Comment by pkozlowski-opensource** [comment]
@maxokorokov what @jdhines says speaks to me. I think that we should move to the native focus indicator _and_ focus _into_ a datepicker only after pressing TAB (so there is no focus indicator initially, after opening a popup)

**Comment by maxokorokov** [comment]
I would very much prefer this for focus/keyboard handling: #1716. I think it would solve this issue too, because navigation arrows would have a separate focus for instance.

I also agree that when you open a month **initially** the focus should not be on the first day, but:
- selected date visible → focus on selected date
- today visible → focus on today
- nothing of the above → maybe focus the first date, maybe focus/highlight nothing until any of the navigation keys are pressed. Not sure, will think about it, needs testing.

**Comment by ar53n** [comment]
@maxokorokov you right. But i think focus on input not should lost. Just need add keyboard button events for select datevalue. But not lost focus from input. Calendar not should be focus

**Comment by shyamal890** [comment]
Totally agree that current date should be highlighted that was the convention in `ui-bootstrap` too. Wonder why it was change in `ng-bootstrap`

**Comment by stesvis** [comment]
ANy news on this? It's a basic requirement IMO

**Comment by mtpultz** [comment]
Any chance the current date being marked will reach v1.1.2?  We can't upgrade to Angular 6 and are stuck using v1.1.2 for our application, but we need a way to have the current date marked on all our datepickers, not just when the datepicker is opened.  

I've looked at a custom day component, but the example is for replacing an input with a component and template when we really just want to extend it for all the datepickers and add a method of `isToday` and use moment to check if `isSame` to apply `is-today` to the classes applied to the day.  Is there an easier way to do this?
